web_src/js/features/user-auth-webauthn.ts

```diff
@@ -5,6 +5,10 @@ import {GET, POST} from '../modules/fetch.ts';
 const {appSubUrl} = window.config;
 
 export async function initUserAuthWebAuthn() {
+  if (!document.querySelector('.user.signin')) {
```

Comment:
This check is not right because `link_account` also uses `signin_inner`

But I guess some logic has been broken for a while.

Reply:
We should probably add a `data-page="user-signin"` attribute to `<body>` at some point to clearly indicate the which page we are on instead of relying on these flimsy selectors.

Reply:
Are you serious? I am talking the logic bug.

Reply:
I guess next time put a negative review, because otherwise, every PR runs danger of being merged too hastily.

And yes, my remark was just some idea for the future, but apparently @techknowlogick had read it incorrectly.